scripts/deploy/README.md

```diff
@@ -16,7 +18,7 @@ IMAGE_DIR=/my/image-dir docker-compose up -d
 At this point, Solr needs to have records for each image in your IMAGE_DIR.
 From the solr directory, run:
 ```
-./import-images.sh SOLR-CONTAINER-NAME imagespace /my/image-dir
+./solr/import-images.sh SOLR-CONTAINER-NAME imagespace /my/image-dir
```

Comment:
This is slightly incompatible from the statement above: "From the solr directory, run", which might imply there are two nested solr dirs. I would change the text above to "At this point, Solr needs to have records for each image in your IMAGE_DIR, which is accomplished by running this import script:"